Tests/HubTests/HubApiTests.swift

```diff
@@ -167,7 +167,7 @@ class HubApiTests: XCTestCase {
 }
 
 class SnapshotDownloadTests: XCTestCase {
-    let repo = "coreml-projects/Llama-2-7b-chat-coreml"
+    var repo = "coreml-projects/Llama-2-7b-chat-coreml"
```

Comment:
Let's move this back to `let` and update `testDownloadSmolLargeFile` to define a new ivar.